android/app-newm/src/main/java/io/newm/LoginActivity.kt

```diff
@@ -84,7 +84,7 @@ class LoginActivity : ComponentActivity() {
                 }
 
                 is Welcome -> ui<WelcomeScreenUiState> { state, modifier ->
-                    WelcomeScreenUi(modifier, state, eventLogger)
```

Comment:
The eventLogger is something an interface we inject in the KMP layer, so in theory if the WelcomeScreenUi is in CMP we should be able to provide it, so far it's looking very good